src/components/admin/branchSettings/BranchFormModal.tsx

```diff
@@ -180,6 +188,25 @@ const BranchFormModal: React.FC<Props> = props => {
                   <Checkbox {...register("secret")}>Secret</Checkbox>
                 </FormControl>
               )}
+              {branchType === "APPLICATION" && secret && (
+                <Popover>
+                  <PopoverTrigger>
+                    <Button>View QR Code</Button>
+                  </PopoverTrigger>
+                  <PopoverContent>
+                    <PopoverArrow />
+                    <PopoverCloseButton />
+                    <PopoverHeader>QR Code</PopoverHeader>
+                    <PopoverBody>
+                      <QRCode
+                        size={256}
+                        style={{ height: "auto", maxWidth: "100%", width: "100%" }}
+                        value={apiUrl(Service.REGISTRATION, `/${hexathonId}/start-application/${props.defaultValues.id}`)}
```

Comment:
Can you have the background of the popup a different color than white? it blends in a lot with the background. Also, can you better center the QR code in the popup?

<img width="525" alt="Screen Shot 2022-10-20 at 1 14 33 AM" src="https://user-images.githubusercontent.com/8273038/196861951-9ffe8168-3745-41ad-a686-f5d568105e6a.png">